src/js/jsx/sections/libraries/assets/AssetPreviewImage.jsx

```diff
@@ -43,30 +43,60 @@ define(function (require, exports, module) {
         
         getInitialState: function () {
             return {
-                loading: true,
-                renditionPath: null
+                loading: false,
+                renditionPath: null,
+                lastModified: null
```

Comment:
I see the null helps with `_updateRenditionPath` starting on init... not really happy with the whole logic here... but oh well!